tools/story.tsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react'
 import Box from '../src'
-import {storiesOf} from '@storybook/react'
+import { storiesOf } from '@storybook/react'
 import allPropertiesComponent from './all-properties-component'
```

Comment:
Not sure if having `allowJs: false` will cause problems with typescript reading this file now?

Reply:
There's an all-properties-component as a tsx file for the stories and a js file for the benchmarks 🙌